### DISCUSSION TITLE
Nested Shape Groups -- do they exist?

### DISCUSSION THREAD

**Discussion by buddhasystem** [mitsuba-renderer/mitsuba3#1234] [Q&A] [answered]
As the subject says -- I see that there is a class for Nested Shapes. Can it be nested itself? This can be important for a few simulation scenarios.

**Comment by njroussel** [comment, marked as answer]
Hi @buddhasystem 

No, that is not supported.
I don't think there is some inherent limitation (OptiX and Embree can support it IIRC), we might not have had a use for it and hence never implemented it.

**Comment by buddhasystem** [comment]
@njroussel  Thank you! I had questions from colleagues - does Mitsuba indeed use Optix?

**Reply by njroussel** [reply]
Yes, it does.

**Reply by buddhasystem** [reply]
@njroussel Is there any sort of licensing issue then?

**Reply by njroussel** [reply]
No? I'm not sure what you're asking.

In any case, I'm not qualified to give legal advice. I'd recommend you post on the OptiX forums if you have any concerns.
